docs/generateAllConfigPartial.java

```diff
@@ -20,6 +20,7 @@
 public class generateAllConfigPartial {
 
     private static Map<String, Option> allConfiguration = new HashMap();
+    private static Set<String> skipProperties = Set.of("registry.auth.url.configured");
```

Comment:
Do you want to hide this configuration in the docs?
Should we add a property to `ConfigProperty` like `hidden`, it would be consistent with CLI frameworks and similar

Reply:
I considered modifying `Info` or `ConfigProperty` or possibly adding something new, but ultimately I thought this would be better, at least for now.  The issue with this property is that it's not one that is generally set by users - it is constructed from other properties that are explicitly set.  So I want to inject it but not document it.  I'm not sure if it's better to annotate that in the code or not.  Seems like a wash to me, and less work to just add the skip list.